assets/src/edit-story/components/panels/layer/layerContent.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+const LayerBackground = styled.span`
+	opacity: .5;
+`;
+
+const LayerText = styled.span`
+	overflow: hidden;
+	white-space: nowrap;
+	text-overflow: ellipsis;
+`;
+
+function LayerContent( { element } ) {
+	switch ( element.type ) {
```

Comment:
Let's move each element into a separate `LayerContent` inside `element/x/layerContent.js` or such. We already do the same pattern for `Display`, `Edit`, `Save`, `Preview`, `TextContent`, etc.

Reply:
Fixed with other stuff in 07f0e2a.